zebra-consensus/src/transaction.rs

```diff
@@ -533,6 +540,26 @@ where
 
         Ok(async_checks)
     }
+
+    /// Verifies a transaction's Orchard shielded data.
+    fn verify_orchard_shielded_data(
```

Comment:
We don't have test coverage here and also in a lot of validation functions in the same file such as `verify_sapling_shielded_data`, `verify_sprout_shielded_data`, `verify_v5_transaction`, `verify_v4_transaction`, etc.

How are we testing this validations ?

Reply:
We spoke about this at the Zebra meeting today:
* Transparent, Sprout, and Sapling code is validated using the mainnet and testnet chains, and some test vectors
* Orchard code needs test vectors from the ECC

There is also a specific follow up ticket for Sapling #2426.
Sprout tests are a low priority because it is deprecated.